README.md

```diff
@@ -44,16 +40,16 @@ your local Maven repository, and runs all of the unit tests:
 The project can be built and deployed on a variety of runtime platforms, but if you want to see it in 
 action as quickly as possible try this:
 
-    $ mvn clean install -Pinstall-all-wildfly8
-    $ cd tools/server-all/target/wildfly-8.2.0.Final/
+    $ mvn clean install -Pinstall-all-wildfly9
```

Comment:
I wasn't too sure about this, simply because of the relative URLs bug with KC. Or is that also in WF9?